docs/build/guides/transactions/send-and-receive-payments.mdx

```diff
@@ -6,1405 +6,574 @@ sidebar_position: 7
-    if (payment.asset_type === "native") {
-      asset = "lumens";
-    } else {
-      asset = payment.asset_code + ":" + payment.asset_issuer;
-    }
-
-    console.log(payment.amount + " " + asset + " from " + payment.from);
-  },
-
-  onerror: function (error) {
-    console.error("Error in payment stream");
-  },
+sendPayments().catch((error) => {
+  console.error("Error executing sendPayments function:", error);
+  process.exit(1);
 });
-
-function savePagingToken(token) {
-  // In most cases, you should save this to a local database or file so that
-  // you can load it next time you stream new payments.
-}
-
-function loadLastPagingToken() {
-  // Get the last paging token from a local database or file
-}
 ```
 
 ```java
-Server server = new Server("https://horizon-testnet.stellar.org");
-KeyPair account = KeyPair.fromAccountId("GC2BKLYOOYPDEFJKLKY6FNNRQMGFLVHJKQRGNSSRRGSMPGF32LHCQVGF");
-
-// Create an API call to query payments involving the account.
-PaymentsRequestBuilder paymentsRequest = server.payments().forAccount(account.getAccountId());
-
-// If some payments have already been handled, start the results from the
-// last seen payment. (See below in `handlePayment` where it gets saved.)
-String lastToken = loadLastPagingToken();
-if (lastToken != null) {
-  paymentsRequest.cursor(lastToken);
-}
-
-// `stream` will send each recorded payment, one by one, then keep the
-// connection open and continue to send you new payments as they occur.
-paymentsRequest.stream(new EventListener<OperationResponse>() {
-  @Override
-  public void onEvent(OperationResponse payment) {
-    // Record the paging token so we can start from here next time.
-    savePagingToken(payment.getPagingToken());
-
-    // The payments stream includes both sent and received payments. We only
-    // want to process received payments here.
-    if (payment instanceof PaymentOperationResponse) {
-      if (!((PaymentOperationResponse) payment).getTo().equals(account.getAccountId()) {
-        return;
-      }
-
-      String amount = ((PaymentOperationResponse) payment).getAmount();
-
-      Asset asset = ((PaymentOperationResponse) payment).getAsset();
-      String assetName;
-      if (asset.equals(new AssetTypeNative())) {
-        assetName = "lumens";
-      } else {
-        StringBuilder assetNameBuilder = new StringBuilder();
-        assetNameBuilder.append(((AssetTypeCreditAlphaNum) asset).getCode());
-        assetNameBuilder.append(":");
-        assetNameBuilder.append(((AssetTypeCreditAlphaNum) asset).getIssuer());
-        assetName = assetNameBuilder.toString();
+// SendPaymentExample.java
+// Java 17, Stellar Java SDK (lightsail-network/java-stellar-sdk)
+
+import org.stellar.sdk.Account;
+import org.stellar.sdk.AssetTypeNative;
+import org.stellar.sdk.KeyPair;
+import org.stellar.sdk.MemoId;
+import org.stellar.sdk.Network;
+import org.stellar.sdk.SorobanServer;
+import org.stellar.sdk.Transaction;
+import org.stellar.sdk.TransactionBuilder;
```

Comment:
Suggest just doing 
```suggestion
import org.stellar.sdk.*;
```
for brevity

Reply:
idiomatic java discourages import splats, main vendor [guidelines](https://cr.openjdk.org/~alundblad/styleguide/index-v6.html#toc-import-statements) state the same. I'd prefer to maintain that convention here for consistency.

Reply:
sucks given the verbosity but sounds good!